setup.py

```diff
@@ -1,324 +1,16 @@
-        join(tmp_src, 'test', 'test_issue492.py'),
-        join(tmp_src, 'test', 'test_issue523.py'),
-        join(tmp_src, 'test', 'test_issue532.py'),
-        join(tmp_src, 'test', 'test_issue535.py'),
-        join(tmp_src, 'test', 'test_issue545.py'),
-        join(tmp_src, 'test', 'test_issue554.py'),
-        join(tmp_src, 'test', 'test_issue563.py'),
-        join(tmp_src, 'test', 'test_issue576.py'),
-        join(tmp_src, 'test', 'test_issue579.py'),
-        join(tmp_src, 'test', 'test_issue604.py'),
-        join(tmp_src, 'test', 'test_issue655.py'),
-        join(tmp_src, 'test', 'test_sparql_agg_distinct.py'),
-        join(tmp_src, 'test', 'test_sparql_agg_undef.py'),
-        join(tmp_src, 'test', 'test_turtle_sort_issue613.py'),
-        join(tmp_src, 'test', 'test_wide_python.py'),
-    ]
-    for fn in six_ed:
-        outfiles_2to3.remove(fn)
-    for fn in outfiles_2to3:
-        print('running 2to3 on', fn)
-
-    util.run_2to3(outfiles_2to3)
 
-    # arrange setup to use the copy
-    sys.path.insert(0, tmp_src)
-
-    return tmp_src
+import os
+from setuptools import setup, find_packages
 
 kwargs = {}
-if sys.version_info[0] >= 3:
-    from setuptools import setup, find_packages
-    # kwargs['use_2to3'] = True  # is done in setup_python3 above already
-    kwargs['install_requires'] = ['isodate', 'pyparsing']
-    kwargs['tests_require'] = ['html5lib', 'networkx']
-    kwargs['requires'] = [
-        'six', 'isodate', 'pyparsing',
-        ]
-    html5lib = 'html5lib'
-    kwargs['src_root'] = setup_python3()
-    assert setup
-else:
-    try:
-        from setuptools import setup, find_packages
-        assert setup
-        kwargs['test_suite'] = "nose.collector"
-        kwargs['install_requires'] = [
-            'six', 'isodate',
-            'pyparsing']
-        kwargs['tests_require'] = ['networkx']
-
-        html5lib = 'html5lib'
-
-    except ImportError:
-        from distutils.core import setup, find_packages
-        html5lib = None
-
-if html5lib is not None: # ie. we're not using the simplified distutils setup
-    kwargs['extras_require'] = {'sparql': ['SPARQLWrapper'], 'html': [html5lib]}
-    kwargs['tests_require'].append('SPARQLWrapper')
-    kwargs['tests_require'].append(html5lib)
-
+kwargs['install_requires'] = [ 'six', 'isodate', 'pyparsing']
+kwargs['tests_require'] = ['html5lib', 'networkx', 'SPARQLWrapper']
+kwargs['test_suite'] = "nose.collector"
+kwargs['extras_require'] = {'sparql': ['SPARQLWrapper'], 'html': ['html5lib']}
 
-# Find version. We have to do this because we can't import it in Python 3 until
-# its been automatically converted in the setup process.
-def find_version(filename):
-    _version_re = re.compile(r'__version__ = "(.*)"')
-    for line in open(filename):
-        version_match = _version_re.match(line)
-        if version_match:
-            return version_match.group(1)
 
-version = find_version('rdflib/__init__.py')
+from rdflib import __version__ as version
```

Comment:
this is bad, if any of the requirements listed above isn't present it will fail... at build time we don't want to `import rdflib`

the reason it doesn't fail in CI is cause we do a `pip install -r requirements.txt` in travis... maybe we shouldn't